### PR TITLE
Bug 1720181 - migration to remove SETA table, v2

### PR DESCRIPTION
**Pull request by Archaeopteryx**
Previous version only removed files.

### CODE REVIEW

File: treeherder/config/settings.py [hunk 1]
```diff
@@ -77,6 +77,7 @@
     'treeherder.log_parser',
     'treeherder.etl',
     'treeherder.perf',
+    'treeherder.seta',
```
Comment (jmaher) [inline treeherder/config/settings.py:80]: why are we adding this back?

File: treeherder/seta/migrations/0001_squashed_0002_remove_task_request_and_jp_timeout.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```
Comment (jmaher) [inline treeherder/seta/migrations/0001_squashed_0002_remove_task_request_and_jp_timeout.py:1]: this file, why are we adding it?  is it needed for a sequence of migrations to work properly?

**Review by jmaher — CHANGES_REQUESTED** [review]
why are empty filestreeherder/seta/migrations/__init__.py and treeherder/seta/models.py created but empty

File: treeherder/config/settings.py (hunk 1 above)
Comment (Archaeopteryx) [reply treeherder/config/settings.py:80]: The app still needs to be alive to run the migration which removes the table. The previous patch didn't contain an amendment which kept it.

File: treeherder/seta/migrations/0001_squashed_0002_remove_task_request_and_jp_timeout.py (hunk 2 above)
Comment (Archaeopteryx) [reply treeherder/seta/migrations/0001_squashed_0002_remove_task_request_and_jp_timeout.py:1]: It's needed for the second migration to work (Django keeps track of migrations).

**Review by jmaher — APPROVED** [review]

**Comment by jmaher** [comment]
@Archaeopteryx feel free to merge this- we will probably need to do more than a regular deploy- I would rather test this on stage and verify there before going to production

**Pull request merged**